miniredis.go

```diff
@@ -340,25 +340,36 @@ func (m *Miniredis) Server() *server.Server {
 	return m.srv
 }
 
+type DumpOptions struct {
+	maxCullLen int
```

Comment:
That will have to be exported: `MaxCullLen`.

Reply:
No longer relevant: using a global var